src/test/scala/edu/ie3/simona/model/thermal/CylindricalThermalStorageSpec.scala

```diff
@@ -0,0 +1,273 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.model.thermal
+
+import edu.ie3.datamodel.models.StandardUnits
+import edu.ie3.datamodel.models.input.thermal.CylindricalStorageInput
+import edu.ie3.simona.test.common.UnitSpec
+import edu.ie3.util.quantities.PowerSystemUnits
+import edu.ie3.util.scala.quantities.KilowattHoursPerKelvinCubicMeters
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.matchers.should.Matchers
+import squants.Energy
+import squants.energy.{KilowattHours, Kilowatts}
+import squants.space.CubicMeters
+import squants.thermal.Celsius
+import tech.units.indriya.quantity.Quantities.getQuantity
+import tech.units.indriya.unit.Units
+
+import java.util.UUID
+
+class CylindricalThermalStorageSpec
+    extends UnitSpec
+    with Matchers
+    with BeforeAndAfterAll {
+
+  final val tolerance: Energy = KilowattHours(1e-10)
```

Comment:
If you change this into an implicit val, you could remove the explicit given tolerances below